user_meaning/custom-recipes/user-meaning/recipe.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
+
+u"""Map constants to user meaning
+
+This plugin was developped by the Ministère de l’Intérieur (French ministry of interior)
+in the context of the program Entrepreneurs d’Intérêt Général 2017
+"""
+
+from concurrent import futures
+import dataiku
+from dataiku.customrecipe import get_input_names_for_role
+from dataiku.customrecipe import get_output_names_for_role
+from dataiku.customrecipe import get_recipe_config
+import json
+import pandas as pd
+
+# We read the addresses from the input dataset
+# And write the coordinates in the output dataset
+input_name = get_input_names_for_role('input')[0]
```

Comment:
this is a bit of a nitpick (and doesn't seem to be done elsewhere in codebase), but it might be a good idea to switch key strings to variables. Although, I guess this is all only important depending on how parameter keys to a plugin are versioned. Anyways, just a suggestion but feel free to ignore